octavia_f5/utils/cert_manager.py

```diff
@@ -33,6 +39,12 @@ def __init__(self):
             invoke_on_load=True,
         ).driver
 
+    @tenacity.retry(
+        retry=tenacity.retry_if_exception_type(
+            (octavia_exc.CertificateRetrievalException)),
```

Comment:
As far as I understand it from the [Tenacity docs](https://tenacity.readthedocs.io/en/latest/api.html?highlight=retry_if_exception_type#tenacity.retry.retry_if_exception_type), this does not have to be a tuple, so you can leave out the parentheses here. But it's okay if you don't, I'm still approving this PR 👍